src/SourceBuild/content/repo-projects/roslyn-analyzers.proj

```diff
@@ -1,9 +1,8 @@
 <Project Sdk="Microsoft.Build.NoTargets">
 
   <PropertyGroup>
-    <LogVerbosityOptOut>true</LogVerbosityOptOut>
-
-    <BuildArgs>$(BuildArgs) $(FlagParameterPrefix)warnAsError $(ArcadeFalseBoolBuildArg)</BuildArgs>
+    <!-- When building source only, dependencies get updated which trigger new nullability warnings. -->
```

Comment:
do we have a tracking issue for that? since eventually we'll see it in the VMR build too

Reply:
The remaining roslyn-analyzers component (NetAnalyzers) will be moved into sdk soon (ETA May-June). The warnings will then show up as part of the move and naturally resolved.